### ISSUE TITLE
Exception in LoadingDataDisplay: Invariant Violation: Objects are not valid as a React child (found: object with keys {}).

### ISSUE THREAD

**Issue by monfresh** [department-of-veterans-affairs/caseflow#11336]
[Sentry alert](https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/5669/).

The offending line is:
```js
return error === COPY.DUPLICATE_PHONE_NUMBER_TITLE ? duplicateNumberTitle : accessDeniedTitle;
```
because both `duplicateNumberTitle` and `accessDeniedTitle` are set to a hash:
```js
const accessDeniedTitle = { title: COPY.ACCESS_DENIED_TITLE };
```
as opposed to similar code where we set it to a String:
```js
return error === COPY.DUPLICATE_PHONE_NUMBER_TITLE ? duplicateNumberMsg : accessDeniedMsg;
```
```js
const accessDeniedMsg = <div>
  It looks like you do not have the necessary level of access to view this information.<br />
  Please check with your application administrator before trying again.</div>;
```

**Comment by monfresh** [comment]
Here's another one in the same file due to `PROMISE_RESULTS` being set to a hash.
https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/4885/events/394572/

**Comment by monfresh** [comment]
@hschallhorn I think you touched this code last. Any ideas here?

**Comment by hschallhorn** [comment]
Wow. Ok so looked into this and I'm at a complete loss as to what's triggering in or even how to reproduce it. Lowell and I also glanced at a few of the other Invariant Violations and sentry has been a total lack of help in diagnosing what's causing them or even where it's getting thrown.

**Comment by jcq** [comment]
Noting that this is still an issue:
https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/7672/

### Tech notes for further troubleshooting
- if the fn fed to `createLoadPromise` prop fails with a status code of 403 or 404, `LoadingDataDisplay` will always work correct
- if it fails with a different status (or w/o one specified), `LoadingDataDisplay` will render out the provided error component (defaults to `StatusMessage`), but add the contents of the `failStatusMessageChildren` prop as children of said error component. Since `failStatusMessageChildren` defaults to an empty object, we end up with the error seen in these sentry alerts if that prop isn't set.

**Comment by rubaiyat22** [comment]
Seeing these in `/hearings/schedule/assign`
https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/7771/?referrer=webhooks_plugin

It looks like it's being caused by 504 error code.

**Comment by yoomlam** [comment]
This alert popped up twice today and reappears regularly: https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/7771/events/

**Comment by pkarman** [comment]
Assigned this to Echo. Reader needs to be adopted. @jimruggiero @alisan16

**Comment by ferristseng** [comment]
Again here: https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/7771/events/1db468cd479a4e91ad578a3c80494bf0/

**Comment by yoomlam** [comment]
May be solved by PR #13353
